native/mod_proxy_cluster/mod_proxy_cluster.c

```diff
@@ -2198,6 +2198,40 @@ static proxy_worker *internal_find_best_byrequests(proxy_balancer *balancer, pro
     return mycandidate;
 }
 
+/* Get the http worker if we are getting a STATUS for ws one. */
+static proxy_worker *get_http_worker(server_rec *s, proxy_worker *ws_worker)
+{
+    if (ws_worker->balancer != NULL) {
+        /* we need to find the corresponding http/https worker */
+        char *ptrw;
+        proxy_balancer *balancer = ws_worker->balancer;
+        int sizew = balancer->workers->elt_size;
+        int j;
+        ptrw = balancer->workers->elts;
+        for (j = 0; j < balancer->workers->nelts; j++,  ptrw=ptrw+sizew) {
```

Comment:
It would be nice to add spaces around operators. Also, it might be cleaner to write `ptrw += sizew`, but it's up to you.